public/app/features/serviceaccounts/components/CreateTokenModal.tsx

```diff
@@ -38,8 +38,11 @@ export const CreateTokenModal = ({ isOpen, token, serviceAccountLogin, onCreateT
   tomorrow.setDate(tomorrow.getDate() + 1);
 
   const maxExpirationDate = new Date();
-  if (config.tokenExpirationDayLimit !== undefined) {
+  if (config.tokenExpirationDayLimit !== undefined && config.tokenExpirationDayLimit > -1) {
+    console.error(`there is a defined date`, config.tokenExpirationDayLimit);
```

Comment:
You probably want to remove this debug log :)

Reply:
What debug log? 😬

jk! removed!